Lib/_pyrepl/windows_console.py

```diff
@@ -407,31 +412,33 @@ def get_event(self, block: bool = True) -> Event | None:
                     continue
                 return None
 
-            key = rec.Event.KeyEvent.uChar.UnicodeChar
+            key_event = rec.Event.KeyEvent
+            raw_key = key = key_event.uChar.UnicodeChar
 
-            if rec.Event.KeyEvent.uChar.UnicodeChar == "\r":
-                # Make enter make unix-like
+            if key == "\r":
+                # Make enter unix-like
                 return Event(evt="key", data="\n", raw=b"\n")
-            elif rec.Event.KeyEvent.wVirtualKeyCode == 8:
+            elif key_event.wVirtualKeyCode == 8:
                 # Turn backspace directly into the command
-                return Event(
-                    evt="key",
-                    data="backspace",
-                    raw=rec.Event.KeyEvent.uChar.UnicodeChar,
-                )
-            elif rec.Event.KeyEvent.uChar.UnicodeChar == "\x00":
+                key = "backspace"
+            elif key == "\x00":
                 # Handle special keys like arrow keys and translate them into the appropriate command
-                code = VK_MAP.get(rec.Event.KeyEvent.wVirtualKeyCode)
+                code = VK_MAP.get(key_event.wVirtualKeyCode)
                 if code:
-                    return Event(
-                        evt="key", data=code, raw=rec.Event.KeyEvent.uChar.UnicodeChar
-                    )
+                    if code in ("left", "right") and (ctrlstate := key_event.dwControlKeyState) and ctrlstate & CTRL_OR_ALT_ACTIVE:
+                        code = f"ctrl {code}"
+                    return Event(evt="key", data=code, raw=key)
                 if block:
                     continue
 
                 return None
 
-            return Event(evt="key", data=key, raw=rec.Event.KeyEvent.uChar.UnicodeChar)
+            if (ctrlstate := key_event.dwControlKeyState) and ctrlstate & ALT_ACTIVE:
+                # first send meta, then send the key
```

Comment:
```suggestion
                # queue the key, return the meta command
```
I find it a bit confusing that the comments says first send meta, but the meta is in the second line (while the key is in the first line)